suggestions/barcode_scanners.json

```diff
@@ -4,5 +4,24 @@
         "label": "Binary Eye",
         "source": "fg",
         "repo": "https://github.com/markusfisch/BinaryEye"
+    },
+    {
+        "id": "com.example.barcodescanner",
+        "label": "QR & Barcode Scanner",
+        "source": "fg",
+        "repo": "https://github.com/wewewe718/QrAndBarcodeScanner"
+    },
+    {
+        "id": "com.atharok.barcodescanner",
```

Comment:
Looks modern with reasonable size. @shuvashish76: Opinions? If everything is right, we can replace **Binary Eye** with it. The UI/UX of Binary Eye hasn't been improved in the past few years.

Reply:
I think the only issue I have with this app is that the Internet features are enabled by default (ie. “Query remote API automatically” is enabled by default) which shouldn't be for any reason. Let me explain a bit in case you haven't actually tried the app:

The app has “Open Food Facts” API enabled by default. It's not like that people only scans for food-related barcode. In a super shop, you can scan barcodes related to food, beauty, music — anything. So, it's not ideal to have it enabled anyway. Besides, after scanning the barcode, there's already an option to fetch results using one of the APIs. So, I don't see why this option should be enabled or query automatically by default. If it's intended to be one of the primary or highlighted features, it should be highlighted/featured during the first run (and prompt user to enable it) instead of enabling it by default.

Reply:
OK. There are a few other minor issues:

1. Backup allowed but it didn't specify the contents to back up
2. No network-security-config present in the manifest (possibly with PKP)
3. Non-exported/private activities aren't marked as so